src/main/java/org/myrobotlab/io/FileIO.java

```diff
@@ -1575,4 +1576,22 @@ public static String getExt(final String filename) {
   }
 
   
+  /**
+   * flips all \ to / or / to \ depending on OS
+   * @param - dirPath non normalized path
+   * @return - fixed path
+   */
+  public static String normalize(String dirPath) {
+    if (dirPath == null) {
+      return null;
+    }
+    Platform platform = Platform.getLocalInstance();
+    if (platform.isWindows()) {
+      return dirPath.replace("/", "\\");

```

Comment:
should be replaceAll ... yes?

Reply:
strangely according to the docs - replace is more appropriate